lib/api/holdings_api.rb

```diff
@@ -46,6 +46,26 @@ class HoldingsAPI < Sinatra::Base
     return_doc.to_json
   end
 
+  # Which organization have holdings that:
+  # * are in the same cluster as item_id and
+  # * has condition: BRT, status: LM, or both
+  # Only report organizations with a count greater than zero.
+  get "/v1/item_held_by_brlm" do
+    # ArgumentError if missing item_id.
+    validate_params(params: params, required: ["item_id"])
+    item_id = params["item_id"]
+
+    ht_item = Clusterable::HtItem.find(item_id: item_id)
+    cluster = ht_item.cluster
+    # cluster.access_counts is a hash of {organization => count}
+    nonzero_access_counts = cluster.access_counts.select { |org, count| count > 0 }
```

Comment:
Let me gripe briefly about `access_counts` as being a horrible name (I know we're at the mercy of the schema). Is there an argument to be made for doing an `alias_method` on it so we can actually mention `brlm` or `brittle_lost_missing` in the code itself?

Reply:
Now would be the time to change it if we want to, and I would agree it's worth considering.

Reply:
I don't have any strong feelings here, and I think this can be easily refactored if/when we decide to add `alias_method :ham_sandwich, :access_counts ` or whatever. So I'm not going to do anything.